templates/projects.html

```diff
@@ -1,3 +1,7 @@
-<h2> Total project Counter {{ counter }} </h2>
+<h2> Total projects {{ counter }} </h2>
 
-{{ projects|safe }}
\ No newline at end of file
+{{ all_projects|safe }}
```

Comment:
What about this:

```suggestion
{% for repo in all_projects %}
<div class="pf-c-card">
    <div class="pf-c-card__header pf-c-title pf-m-md">{{repo}}</div>
    <div class="pf-c-card__body">https://project.url and other info</div>
</div>
{% endfor %}
```

You need to remove `str` conversion in `render_template` call.

Reply:
btw, we have some probably some "{repository_name}" value in redis:

![Screenshot from 2020-01-05 12-04-50](https://user-images.githubusercontent.com/20214043/71778876-b3c67280-2fb3-11ea-8383-9bb58e5f9120.png)